cmd/nomos/vet/vet.go

```diff
@@ -75,15 +77,29 @@ returns a non-zero error code if any issues are found.
   nomos vet --path=my/directory
   nomos vet --path=/path/to/my/directory`,
 	Args: cobra.ExactArgs(0),
+	PreRunE: func(cmd *cobra.Command, _ []string) error {
+		if flags.SkipAPIServer && len(flags.NoAPIServerCheckForGroup) > 0 {
+			// If --no-api-server-check is specified, ignore --no-api-server-check-for-group.
```

Comment:
I believe in the design doc it was decided to return an error if both were specified